### ISSUE TITLE
bug(YouTube - Enable slide to seek): Including patch causes issues with clicking and double tapping the video player controls

### ISSUE THREAD

**Issue by SeokHyeonLee** [ReVanced/revanced-patches#2439]
### Bug description

revanced-patches : v3.1.0 (also on v2.202.0)
revanced-integrations : v1.0.0
revanced-manager : v1.17.3
revanced-cli : v4.3.0

android version : 9.0(Galaxy Note 8), 11.0 (Galaxy S21U)

The YouTube player is having issues with the pause/play button (player menu buttons) and double-tapping to seek because the clicks are stuck when not zoomed in.
If you zoom 1.1x (but not 1.0x), then the pause/play button and double-tapping to seek will work as normal.

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by oSumAtrIX** [comment]
Disable the slide to seek patch or it's option and check if that make a difference

**Comment by SeokHyeonLee** [comment]
> Disable the slide to seek patch or it's option and check if that make a difference

already disable slide to seek and swipe controls
all combinations of  settings related to interactions are stucked

**Comment by oSumAtrIX** [comment]
Follow this and report back:

![Screenshot_20231215-085450](https://github.com/ReVanced/revanced-patches/assets/13122796/d4c2df01-b2f9-462a-9bf4-c0c73413cb40)

**Comment by KobeW50** [comment]
I believe this has been confirmed by several users that it is indeed the `Enable slide to seek` patch which causes this, so the "waiting on author" can be removed and perhaps the title can be updated.

**Comment by oSumAtrIX** [comment]
The patch should be disabled with a warning or removed at this point in this case.
